Doc/library/email.utils.rst

```diff
@@ -25,7 +25,8 @@ module:
     is in effect for the specified time.
 
     .. versionadded:: 3.3
-
+    .. deprecated:: 3.11
```

Comment:
I think we need a newline between these two:

```suggestion

    .. deprecated:: 3.11
```

Reply:
Why not using `.. deprecated-removed:: 3.11 3.13` instead?